rllib/evaluation/rollout_worker.py

```diff
@@ -1252,6 +1252,9 @@ def add_policy(
 
         self.filters[policy_id] = get_filter(self.observation_filter, filter_shape)
 
+        if self.policy_config.get("enable_connectors") and policy_id in self.policy_map:
```

Comment:
can you check if self.policy_map[policy_id] already has connectors configured?
when you restore a connector-enabled policy, existing connector state is part of the policy state, so it should be restore already, and we don't need to create things from fresh here.
and I think the logic right now is that you can't really restore non-connector policies for a connector-enabled stack.

Reply:
Done!